src/pynxtools_raman_multiformat/reader.py

```diff
@@ -129,18 +229,17 @@ def read_txt_file(self, filepath):
         # Transform: [[A, B], [C, D], [E, F]] into [[A, C, E], [B, D, F]]
         data = [list(item) for item in zip(*data)]
 
-        #transform linewise read data to colum style data
-        data = np.transpose(data)
-
         # assign column data with keys
         data_dict = {
-            "data/x_values": data[:, 0],
-            "data/y_values": data[:, 1]
+            "data/x_values": data[0],
+            "data/y_values": data[1]
         }
 
-        self.txt_data = data_dict
-        self.txt_header = header_dict
 
+        self.raman_data = data_dict
+        self.post_process = post_process_witec.__get__(self, RamanReaderMulti)
+
+        return {}
 
     def get_eln_data(self, key: str, path: str) -> Any:

```

Comment:
Also remive here (or only keep one at least)

Reply:
yea, deleted one of the two get_eln_data functions